Assets/Stack mini game/Scripts/StackMinigameUI.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StackMinigameUI : MonoBehaviour
+{
+    [SerializeField] private GameObject stackButton;
+    [SerializeField] private GameObject replayButton;
+
+    [SerializeField] private Text outcomeText;
+
+
+    public void SetGameUI(GameState gameState)

```

Comment:
I would say a switch statement would make more sense here. 

This looks like control flow, when in reality it's just matching one of the 3 finite states. 

It is functionally the same, but would be clearer if it switched the value, imo.